FirebaseRemoteConfig/Sources/FIRRemoteConfig.m

```diff
@@ -329,6 +329,8 @@ - (void)activateWithCompletion:(FIRRemoteConfigActivateChangeCompletion)completi
     // New config has been activated at this point
     FIRLogDebug(kFIRLoggerRemoteConfig, @"I-RCN000069", @"Config activated.");
     [strongSelf->_configContent activatePersonalization];
+    [strongSelf->_settings updateLastActiveTemplateVersion];
```

Comment:
Thinking: We don't need to wait for this to complete before calling `activateRolloutMetadata` since UserDefaults is updated synchronously in the process (and persisted to disk async) so rollouts will get the latest value in the process https://developer.apple.com/documentation/foundation/userdefaults#overview

Reply:
Make sense to me. Dispatch the activeTemplateVersion updating.

Reply:
Ah, I was saying I think that the previous code (synchronously) is fine, since UserDefaults is updated in the process synchronously. Can revisit this when we use the value in a subsequent PR, though